data/campaigns/World_Conquest/resources/data/training.cfg

```diff
@@ -9,6 +9,7 @@
         image=attacks/curse.png
         name=_ "Dark"
         dialogue=_ "I haven’t lived centuries fighting others’ battles, but I could combat boredom teaching your men a couple of things about dark techniques."
+        manual_invest = False
```

Comment:
Why is the F capitalized?

Reply:
Bad Python habits on my behalf. I updated "False" to be "false".

Reply:
in wml, that would be `no`

Reply:
There are three other uses of `false` in that file though, so presumably it still works?

Reply:
I changed it to "no", not seeing this comment. I am happy to flip it back, whichever is preferred.

Reply:
`true`/`false` is not wrong just `yes`/`no` is a bit more canonical. Since in WC it seems more the other way around it doesn't matter much either way.

Reply:
I flipped it back to true / false simply because it was erroring the other way; and, it is consistent with the rest of WC.